src/ports/postgres/modules/recursive_partitioning/decision_tree.sql_in

```diff
@@ -355,6 +355,19 @@ tree_train(
     <th>independent_var_types</th>
     <td>TEXT. A comma separated string for the types of independent variables.</td>
     </tr>
+
+    <tr>
+    <th>n_folds</th>
+    <td>BIGINT. Number of cross-validation folds used.</td>
+    </tr>
+
+    <tr>
+    <th>null_proxy</th>
+    <td>TEXT. Describes how NULLs are handled.  If NULL is not 
+    treated as a separate categorical variable, this will be blank.
```

Comment:
I suggest replacing `this will be blank` with `this will be NULL`. The `blank` for NULL is the default in psql but that can easily be changed.